samples/Wren/ex.wren

```diff
@@ -0,0 +1,15 @@
+System.print("Hello, world!")
+
```

Comment:
Linguist needs real-world usage examples to properly train the classifier. "Hello world" code is _never_ real-world usage code. Please replace this with an example of a file from a real project and provide a reference to it and it's license in a comment in the PR.

Reply:
This might be a good sample file to use: https://github.com/munificent/wrenalyzer/blob/master/parser.wren

MIT license: https://github.com/munificent/wrenalyzer/blob/master/LICENSE